### DISCUSSION TITLE
Login not possible anymore - anything I can do?

### DISCUSSION THREAD

**Discussion by kaleidoscube** [requarks/wiki#5072] [Error / Bug Report]
After adding Google authentication I cannot login to the wiki anymore - not even with the admin user. 

![image](https://user-images.githubusercontent.com/100562356/156802541-6f452117-a6cf-4921-a71d-cabe1eaf8308.png)

When I login with the admin, it says:
![image](https://user-images.githubusercontent.com/100562356/156802628-fd483822-7abe-4549-9d6a-28d0aade23a9.png)

However, after that I just land on the login page again, with nothing else to do. This pretty much locked me out of the wiki, same thing happens when I use the newly created google login.  

Is there a way I can reset users with docker to start over again (on the user side?) As the wiki is private, it would be a shame if all of our documents would be lost.

**Comment by Phil-hostwiki** [comment]
Have you checked the logs to know what the problem might be?
